<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HyoNumberPad</class>
 <widget class="QDialog" name="HyoNumberPad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>210</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>160</width>
    <height>210</height>
   </size>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="QLineEdit" name="lineEdit">
     <property name="maximumSize">
      <size>
       <width>142</width>
       <height>27</height>
      </size>
     </property>
     <property name="maxLength">
      <number>32</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QToolButton" name="toolButton_2">
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>27</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="toolButton_Right">
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>27</height>
      </size>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QToolButton" name="toolButton_0">
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>27</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QToolButton" name="toolButton_1">
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>27</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QToolButton" name="toolButton_3">
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>27</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QToolButton" name="toolButton_4">
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>27</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QToolButton" name="toolButton_5">
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>27</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QToolButton" name="toolButton_6">
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>27</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QToolButton" name="toolButton_7">
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>27</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="toolButton_8">
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>27</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="toolButton_9">
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>27</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="toolButton_Backspace">
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>27</height>
      </size>
     </property>
     <property name="text">
      <string>BS</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QToolButton" name="toolButton_Cancel">
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>27</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ESC</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QToolButton" name="toolButton_Clear">
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>27</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CLR</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QToolButton" name="toolButton_Delete">
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>27</height>
      </size>
     </property>
     <property name="text">
      <string>DEL</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3" rowspan="2">
    <widget class="QToolButton" name="toolButton_Enter">
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ENT</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="toolButton_Left">
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>27</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QToolButton" name="toolButton_Minus">
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>27</height>
      </size>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QToolButton" name="toolButton_Point">
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>27</height>
      </size>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
